vscode/package.json

```
@@ -159,6 +165,13 @@
           "scope": "machine",
           "order": 0
         },
+        "konveyor.rpcServerPath": {
```

Sorry I missed this the first time around.
```suggestion
        "konveyor.kaiRpcServerPath": {
```